src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -458,8 +458,8 @@ private static LogEntry WriteEntry(EventLogLevel eventLogLevel, Exception innerE
             // Prefix header (library version, DateTime, whether PII is ON/OFF) to the first message logged by Wilson.
             if (!_isHeaderWritten)
             {
-                string headerMessage = string.Format(CultureInfo.InvariantCulture, "Microsoft.IdentityModel Version: {0}. Date {1}. {2}",
-                    typeof(IdentityModelEventSource).Assembly.GetName().Version.ToString(),
+                string headerMessage = string.Format(CultureInfo.InvariantCulture!, "Microsoft.IdentityModel Version: {0}. Date {1}. {2}",
```

Comment:
why is this needed?

Reply:
+1 should not be needed

Reply:
It was throwing an error before but seems to not be the case anymore. Will remove thanks!